napari/_qt/qt_viewer.py

```diff
@@ -728,10 +728,10 @@ def _open_folder_dialog(self):
         dlg.setHistory(hist)
 
         folder = dlg.getExistingDirectory(
-            parent=self,
-            caption=trans._('Select folder...'),
-            directory=hist[0],  # home dir by default
-            options=(
+            self,
```

Comment:
Is there a reason you removed the keywords for the arguments? I can usually stomach 1 and sometimes 2 (depending on the function) positional arguments from a readability standpoint.